packages/dds/tree/src/test/feature-libraries/sequence-field/sequenceFieldEditor.test.ts

```diff
@@ -26,19 +27,19 @@ export function testEditor() {
 		});
 
 		it("insert one node", () => {
-			const actual = SF.sequenceFieldEditor.insert(42, 1, id);
+			const actual = SF.sequenceFieldEditor.insert(42, 1, id, mintRevisionTag());
```

Comment:
These tests shouldn't be passing. The `expected` changeset needs to have the same revision as the one passed in the editor.